torch/fx/graph_module.py

```diff
@@ -706,6 +706,24 @@ def __deepcopy__(self, memo):
     def __copy__(self):
         return GraphModule(self, self.graph)
 
+    def nested_code(self) -> str:
```

Comment:
Hmm, I'm not sure I want to call it `nested_code`, since you can't actually eval() this. Maybe just call it `nested_str`?

Reply:
You should add `@compatibility(is_backward_compatible=False)`